pyrate/merge.py

```diff
@@ -272,44 +291,37 @@ def _save_merged_files(ifgs_dict, outdir, array, out_type, index=None, savenpy=N
         dest = join(outdir, out_type + "_" + str(epoch) + ".tif")
         npy_file = join(outdir, out_type + "_" + str(epoch) + ".npy")
         # sequence position; first time slice is #0
-        md['SEQUENCE_POSITION'] = index+1
+        md['SEQUENCE_POSITION'] = index + 1
         md[ifc.EPOCH_DATE] = epoch
     else:
         dest = join(outdir, out_type + ".tif")
         npy_file = join(outdir, out_type + '.npy')
         md[ifc.EPOCH_DATE] = [d.strftime('%Y-%m-%d') for d in epochlist.dates]
 
-    if out_type == 'stack_rate':
-        md[ifc.DATA_TYPE] = ifc.STACKRATE
-    elif out_type == 'stack_error':
-        md[ifc.DATA_TYPE] = ifc.STACKERROR
-    elif out_type == 'stack_samples':
-        md[ifc.DATA_TYPE] = ifc.STACKSAMP
-    elif out_type == 'linear_rate':
-        md[ifc.DATA_TYPE] = ifc.LINRATE
-    elif out_type == 'linear_error':
-        md[ifc.DATA_TYPE] = ifc.LINERROR
-    elif out_type == 'linear_samples':
-        md[ifc.DATA_TYPE] = ifc.LINSAMP
-    elif out_type == 'linear_intercept':
-        md[ifc.DATA_TYPE] = ifc.LINICPT
-    elif out_type == 'linear_rsquared':
-        md[ifc.DATA_TYPE] = ifc.LINRSQ
-    elif out_type == 'tsincr':
-        md[ifc.DATA_TYPE] = ifc.INCR
-    elif out_type == 'tscuml':
-        md[ifc.DATA_TYPE] = ifc.CUML
-    else:
-        log.warning('Output type "{}" not recognised'.format(out_type))
+    md[ifc.DATA_TYPE] = out_type_md_dict[out_type]
+
+    if out_type in los_projection_out_types:  # apply LOS projection for these outputs
+        incidence_path = Path(Configuration.geometry_files(params)['incidence_angle'])
+        if incidence_path.exists():  # We can do LOS projection
+            if params[C.LOS_PROJECTION] == ifc.LINE_OF_SIGHT:
+                log.info(f"Retaining Line-of-sight signal projection for file {dest}")
+            elif params[C.LOS_PROJECTION] != ifc.LINE_OF_SIGHT:
+                log.info(f"Projecting Line-of-sight signal into {ifc.LOS_PROJECTION_OPTION[params[C.LOS_PROJECTION]]} for file {dest}")
+            incidence = shared.Geometry(incidence_path)
+            incidence.open()
+            array /= los_projection_divisors[params[C.LOS_PROJECTION]](incidence.data)
```

Comment:
From the other comment, I think this needs to be:
`array *= los_projection_multipliers[params[C.LOS_PROJECTION]](incidence.data)`

Reply:
My comment is incorrect. The code is implemented correctly